prow/plugins/approve/approve_test.go

```diff
@@ -1372,6 +1372,1991 @@ Approvers can cancel approval by writing ` + "`/approve cancel`" + ` in a commen
 	}
 }
 
+func TestHandle_GanularApprover(t *testing.T) {
```

Comment:
>It is difficult to tell that granular still handles everything simple does, and also to tell what granular handles that simple does not.
>
>It would be better if we could abstract to have a common testApprove(T, handler) type driver, or at least some commonality in fixtures

This still needs addressing